codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -307,11 +448,29 @@ public SubTypeOne polymorphicTypeWithoutSubTypes() {
 
     /**
      * Returns the value of the EnumType property for this object.
+     * <p>
+     * If the service returns an enum value that is not available in the current SDK version, {@link #enumType} will
+     * return {@link EnumType#UNKNOWN}. The raw value returned by the service is available from {@link #enumTypeString}.
```

Comment:
We should use SDK_UNKNOWN or something like that to prevent clash with service define values

https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-models/src/main/resources/models/discovery-2015-11-01-model.json#L425

Reply:
+1